.cicd/scripts/srw_ftest.sh

```diff
@@ -115,16 +120,25 @@ export OMP_NUM_THREADS=1
 )
 set +x
 
-echo "# Try the first few simple SRW tasks ..."
 results_file=${workspace}/functional_test_results_${SRW_PLATFORM}_${SRW_COMPILER}.txt
 rm -f ${results_file}
+
 status=0
-for task in ${TASKS[@]:0:${TASK_DEPTH}} ; do
+
+# Limit to machines that are fully ready
+deny_machines=( gaea hera )
+#sed "s|ulimit -s unlimited;|ulimit -S -s unlimited;|" -i ${workspace}/ush/machine/hera.yaml
```

Comment:
Change lines 129-130 to include hera:

```
deny_machines=( gaea )
sed "s|ulimit -s unlimited;|ulimit -S -s unlimited;|" -i ${workspace}/ush/machine/hera.yaml
```
and line 63   in ./scripts/exregional_make_grid.sh to the following (remove curly brackets):

eval $PRE_TASK_CMDS

Reply:
@natalie-perlin and @BruceKropp-Raytheon -

I've found another way to make the `srw_ftest.sh` script run on Hera that doesn't require removing the curly brackets from `${PRE_TASK_CMDS}` in `scripts/exregional_make_grid.sh`.

When I moved line 130:

`sed "s|ulimit -s unlimited;|ulimit -S -s unlimited;|" -i ${workspace}/ush/machine/hera.yaml`

to line 89 (this line can be moved to any line before the `.generate_FV3LAM_wflow.py` line), the wrapper script successfully ran on Hera.

```
@@ -86,6 +86,8 @@ set -e -u
 
 export PYTHONPATH=${workspace}/ush/python_utils/workflow-tools:${workspace}/ush/python_utils/workflow-tools/src
 
+sed "s|ulimit -s unlimited;|ulimit -S -s unlimited;|" -i ${workspace}/ush/machine/hera.yaml
+
 cd ${workspace}/ush
         # Consistency check ...
         #./config_utils.py -c ./config.yaml -v ./config_defaults.yaml -k "(\!rocoto\b)"

```
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   .cicd/scripts/srw_ftest.sh

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	build_intel/
	tests/build_test244350.out

no changes added to commit (use "git add" and/or "git commit -a")

```
```
# Try hera with the first few simple SRW tasks ...
./run_make_grid.sh ... COMPLETE
./run_get_ics.sh ... COMPLETE
./run_get_lbcs.sh ... COMPLETE
./run_make_orog.sh ... COMPLETE

```
I think that this modification would be better than removing the curly brackets in an ex-script.  If you could try making this modification and double-checking that it will run manually, I would be greatly appreciative.  Thanks!